packages/@ama-sdk/core/src/plugins/timeout/timeout.fetch.ts

```diff
@@ -133,7 +133,7 @@ export class TimeoutFetch implements FetchPlugin {
             if (timer) {
               clearTimeout(timer);
             }
-            this.timerSubscription = this.timerSubscription.filter(callback => timerCallback !== callback);
+            this.timerSubscription = this.timerSubscription.filter((callback) => timerCallback !== callback);
```

Comment:
I am not a fan of this one :) I like more the opposite :D

Reply:
rule `@stylistic/arrow-parens`